blockchain/chain_sync/Cargo.toml

```diff
@@ -9,6 +9,7 @@ anyhow.workspace                 = true
 async-std                        = { workspace = true, features = ["tokio1", "unstable"] }
 async-trait.workspace            = true
 cid                              = { workspace = true, default-features = false, features = ["std"] }
+flume                            = "0.10"
```

Comment:
Given that we also have this defined in `node`, maybe we could move the version to parent workspace?

Reply:
yes, moved